modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/ObjectEntriesFDSPropsTransformer.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+interface ObjecEntriesDataRendererProps {
+	value: {
```

Comment:
@Artartur Please, rename this to `values`, since it's an array of the defined objects.

Reply:
Done!